### PR TITLE
Revert the PR 1744 as the yaml files have been added back

### PR DESCRIPTION
**Pull request by lei-tang**
The PR 1744 fixes the non-existent yaml files in the description of Citadel health checking in
https://preliminary.istio.io/docs/tasks/security/health-check.html reported by the issue https://github.com/istio/istio/issues/6922. Since the PR https://github.com/istio/istio/pull/7178 adds back the removed yaml files, the PR 1744 is reverted.

### CODE REVIEW

File: content/docs/tasks/security/health-check/index.md [hunk 1]
```diff
@@ -115,11 +130,16 @@ continuously failed health checks.
 
 ## Cleanup
 
-*   To disable health checking on Citadel, deploy Citadel with health checking disabled by setting the Helm argument `security.healthCheckEnabled` as `false`.
+*   To disable health checking on Citadel:
+
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
+    $ kubectl delete svc istio-citadel -n istio-system
```
Comment (myidpt) [inline content/docs/tasks/security/health-check/index.md]: We don't need to delete the service. The service is enabled by default now.

File: content/docs/tasks/security/health-check/index.md [hunk 2]
```diff
@@ -115,11 +130,16 @@ continuously failed health checks.
...
+    {{< /text >}}
 
-*   To remove Citadel deployment and service:
+*   To remove Citadel:
 
     {{< text bash >}}
-    $ kubectl delete deploy istio-citadel -n istio-system
+    $ kubectl delete -f install/kubernetes/istio-citadel-with-health-check.yaml
     $ kubectl delete svc istio-citadel -n istio-system
```
Comment (myidpt) [inline content/docs/tasks/security/health-check/index.md]: Same here. Please remove this line.

File: content/docs/tasks/security/health-check/index.md [hunk 3]
```diff
@@ -80,13 +91,17 @@ Observe that the health checking interval is about 15 seconds, which is the defa
...
+`install/kubernetes/istio-citadel-with-health-check.yaml`, and locate the following lines.
 
 {{< text plain >}}
 ...
   - --liveness-probe-path=/tmp/ca.liveness # path to the liveness health checking status file
   - --liveness-probe-interval=60s # interval for health checking file update
   - --probe-check-interval=15s    # interval for health status check
+  - --logtostderr
+  - --stderrthreshold
+  - INFO
```
Comment (myidpt) [inline content/docs/tasks/security/health-check/index.md]: So I guess Citadel can't work well without these lines? If we need to add these lines, we should modify our yaml file template.

File: content/docs/tasks/security/health-check/index.md (hunk 1 above)
Comment (lei-tang) [reply content/docs/tasks/security/health-check/index.md]: Done: no deleting the service.

File: content/docs/tasks/security/health-check/index.md (hunk 2 above)
Comment (lei-tang) [reply content/docs/tasks/security/health-check/index.md]: Done.

File: content/docs/tasks/security/health-check/index.md (hunk 3 above)
Comment (lei-tang) [reply content/docs/tasks/security/health-check/index.md]: To be consistent with the yaml file, these lines have been removed.

File: content/docs/tasks/security/health-check/index.md [hunk 4]
```diff
@@ -115,11 +127,14 @@ continuously failed health checks.
 
 ## Cleanup
 
-*   To disable health checking on Citadel, deploy Citadel with health checking disabled by setting the Helm argument `security.healthCheckEnabled` as `false`.
```
Comment (sdake) [inline content/docs/tasks/security/health-check/index.md]: this content should not be lost.  Please revert this part of the change.

Comment (sdake) [reply content/docs/tasks/security/health-check/index.md]: @ymesika my bad, thought we did :)  It must have been removed.

/lgtm
/approve

**Comment by sdake** [comment]
Just noticed the linter failed in many areas - can you resolve?

TIA
-steve

**Pull request merged**